fakestorage/object.go

```diff
@@ -105,6 +119,10 @@ func toBackendObjects(objects []Object) []backend.Object {
 			Crc32c:      o.Crc32c,
 			Md5Hash:     o.Md5Hash,
 			ACL:         string(o.ACL),
+			Created:     getCurrentIfZero(o.Created).Format(time.RFC3339),
```

Comment:
It was easier/natural to keep delete updated in the backend rather than here, but the backend is "silly" regarding creation/update times. We can move that there too if you think is convenient